atmosphere.py

```diff
@@ -0,0 +1,34 @@
+class Atmosphere:
+  """
+  The atmosphere class will set parameters for the environment in which the rocket will operate.
+  Additional parameters and increasing complexity can be implemented to better simulate real world conditions.
+  Units will be metric for consistency.
+
+  Current parameters include:
+  altitude: h
+  density: rho(h)
+  gravity: g(h)
+  pressure: P(h)
+  temperature: T(h)
+  viscosity: mu(h)
+  """
+
+  def __init__(self, altitude, density, gravity, pressure, temperature, viscosity):
+    self.altitude = altitude  # m
+    self.density = density  # kg/m^3
+    self.gravity = gravity  # m/s^2
+    self.pressure = pressure  # kg/m/s^2
+    self.temperature = temperature  # C
+    self.viscosity = viscosity  # m^2/s
+
+
+# Example: constant, ideal atmosphere
+import numpy as np
+
+h = np.arange(0, 100000, 1000)  # m         sea level to the Karman Line (edge of space)
```

Comment:
Let's throw this is in a main function so you can execute it

def main():
  <logic>

if __name__ == '__main__':
  main()